include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -731,6 +731,7 @@ def TT_ReduceOp: TT_Op<"reduce",
       llvm::SmallVector<RankedTensorType> getInputTypes();
       llvm::SmallVector<Type> getElementTypes();
       unsigned getNumOperands();
+      ::mlir::Operation *getSingleCombiner();
```

Comment:
Can you add documentation for this function too? Something like "Returns the combiner op iff this reduction's region contains only one combiner op other than the return. Returns nullptr otherwise."